src/core/packages/chrome/browser-internal/src/chrome_service.tsx

```diff
@@ -405,9 +405,8 @@ export class ChromeService {
       projectNavigation.setProjectHome(homeHref);
     };
 
-    const setProjectName = (projectName: string) => {
-      validateChromeStyle();
```

Comment:
Seems related to my other [comment](https://github.com/elastic/kibana/pull/238078/files#r2454734225).
Does this mean that everybody can now set the text of the first breadcrumb?
Not saying it's necessarily a bad thing, goes in the lines of exposing the setter publicly in Chrome's API.